src/components/numeral-date/numeral-date.component.tsx

```diff
@@ -383,8 +415,15 @@ export const NumeralDate = <DateType extends NumeralDateObject = FullDate>({
                       onChange={(e) =>
                         handleChange(e, datePart as keyof NumeralDateObject)
                       }
-                      inputRef={(ref) => {
-                        refs.current[index] = ref;
+                      ref={(element) => {
+                        refs.current[index] = { current: element };
```

Comment:
```suggestion
                        refs.current[index] = ref;
```

This change would require following change on line 308

`(ref) => ref === document.activeElement`

Reply:
I don't think I understand what you're suggesting here. `ref` does not actually exist in scope here after my changes. The whole point of this change was to replace `inputRef`, an internal callback prop which accepts a ref object (whose "current" property is a DOM element, or null), with a standard React callback ref, which takes the DOM element itself (or null) as argument. And because of how `refs` is used, each element of the `refs.current` array has to be a "ref object", hence setting it to `{ current: element }`, where before it was `ref` - they're the same shape, but I now have to build the object myself rather than just getting it from the argument.

I'm not sure what you're suggesting (given that the literal change you suggest would cause an error) but hopefully the above gives more context so you can explain it in a way I can understand :)

Reply:
Sorry it should have been:

```suggestion
                        refs.current[index] = element;
```

Right now it looks like we're unnecessarily wrapping the `element` in ref-like `{ current: element }` object.
And then on the line 308, we're accessing it this `(ref) => ref.current === document.activeElement`.
It seems that this one additional level of nesting is unnecesary.